diff -ruN aprx-2.8.2.orig/Makefile.in aprx-2.8.2/Makefile.in
--- aprx-2.9.0.orig/Makefile.in	2015-04-06 01:23:41.000000000 +0200
+++ aprx-2.9.0/Makefile.in	2015-12-26 14:12:24.231772254 +0100
@@ -94,10 +94,10 @@
 
 
 $(PROGAPRX):	$(OBJSAPRX) VERSION Makefile
-		$(LD) $(LDFLAGS) -o $@ $(OBJSAPRX) $(LIBS)
+		$(CC) $(CCFLAGS) -o $@ $(OBJSAPRX) $(LIBS)
 
 $(PROGSTAT):	$(OBJSSTAT) VERSION Makefile
-		$(LD) $(LDFLAGS) -o $@ $(OBJSSTAT) $(LIBS)
+		$(CC) $(CCFLAGS) -o $@ $(OBJSSTAT) $(LIBS)
 
 .PHONY:		man
 man:		$(MAN)
