diff -ruN aprx-2.8.2.orig/Makefile.in aprx-2.8.2/Makefile.in
--- aprx-2.8.2.orig/Makefile.in	2017-01-16 00:31:58.243085783 +0100
+++ aprx-2.8.2/Makefile.in		2017-01-16 00:31:19.454990431 +0100
@@ -93,10 +93,10 @@
 
 
 $(PROGAPRX):	$(OBJSAPRX) VERSION Makefile
-		$(LD) $(LDFLAGS) -o $@ $(OBJSAPRX) $(LIBS)
+		$(CC) $(CCFLAGS) -o $@ $(OBJSAPRX) $(LIBS)
 
 $(PROGSTAT):	$(OBJSSTAT) VERSION Makefile
-		$(LD) $(LDFLAGS) -o $@ $(OBJSSTAT) $(LIBS)
+		$(CC) $(CCFLAGS) -o $@ $(OBJSSTAT) $(LIBS)
 
 .PHONY:		man
 man:		$(MAN)
